### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by Serial-ATA**
This is mostly redundant code removal with a few fixes sprinkled in. It's really just an attempt to make the IDE complain a little less :smile:.

I wasn't sure if any of these were intentional, so I split each area up into a separate commit.

### CODE REVIEW

File: vane-core/src/main/java/org/oddlama/vane/core/data/DurabilityOverrideData.java [hunk 1]
```diff
@@ -188,7 +188,7 @@ protected boolean is_durability_lore(final Component component) {
 	private int damage_clamp(final int fake_damage, final int fake_max, final float dangerous_interpolated_damage, final short max) {
 		// This can only happen in weird edgecases, in which case we recover by breaking the item the next time
 		// the player uses it. Much better then showing negative values in the fake durability lore.
-		if (fake_damage >= fake_max) return max - 1;
+		if (fake_damage > fake_max) return max - 1;
```
Comment (oddlama) [inline vane-core/src/main/java/org/oddlama/vane/core/data/DurabilityOverrideData.java:191]: I'm not sure whether this change breaks something. This is one of the few things that weren't written by me, so we have to find out whether that change is fine.

I just saw that before, `max-1` would be returned if `fake_damage == fake_max`, but now we get `max`. Essentially, the if statement below this one was dead code before.

Comment (oddlama) [reply vane-core/src/main/java/org/oddlama/vane/core/data/DurabilityOverrideData.java:191]: Oh wait, nevermind. It seems like I already rewrote this code and the whole `DurabilityOverrideData.java` class isn't used (the code is now all in DurabilityManager.java). I just forgot to remove it when I did that :D

So instead, please just delete the whole file! 😄

File: vane-trifles/src/main/java/org/oddlama/vane/trifles/ChestSorter.java [hunk 2]
```diff
@@ -166,42 +166,29 @@ public void on_player_right_click(final PlayerInteractEvent event) {
...
-			switch (facing) {
-				case NORTH:
-				case SOUTH:
-					rx = config_radius_x;
-					rz = config_radius_z;
-					break;
-				case EAST:
-				case WEST:
-					rx = config_radius_z;
-					rz = config_radius_x;
```
Comment (oddlama) [inline vane-trifles/src/main/java/org/oddlama/vane/trifles/ChestSorter.java:184]: Careful, this is different from the refactored code. X=Z here, but X=X below. That's different for all assignments. I think both switches must stay...

File: vane-trifles/src/main/java/org/oddlama/vane/trifles/items/File.java [hunk 3]
```diff
@@ -342,7 +342,7 @@ private Sound change_wall(final Player player, final Block block, final Wall wal
 				if (target_height == Wall.Height.NONE) {
 					target_height = Wall.Height.LOW;
 				}
-				wall.setHeight(adjusted_clicked_face, Wall.Height.LOW);
+				wall.setHeight(adjusted_clicked_face, target_height);
```
Comment (oddlama) [inline vane-trifles/src/main/java/org/oddlama/vane/trifles/items/File.java:345]: Thanks, I guess that would have qualified as a BUG 😄

**Review by oddlama — CHANGES_REQUESTED** [review]
Except for one thing everything looks good! Thanks!

File: vane-trifles/src/main/java/org/oddlama/vane/trifles/ChestSorter.java (hunk 2 above)
Comment (Serial-ATA) [reply vane-trifles/src/main/java/org/oddlama/vane/trifles/ChestSorter.java:184]: Oh wow, good catch! I looked over this multiple times thinking that was going to be the case and never caught that :sweat_smile:.

**Comment by Serial-ATA** [comment]
Noticed one more thing :)

**Review by oddlama — APPROVED** [review]

**Pull request merged**